DEVELOPMENT.md

```diff
@@ -2,6 +2,20 @@ Tips for developers
 -------------------
 It's dangerous to go alone. Take these tips in case you need to fit Indico to your particular needs.
 
+## Running a virtual machine
+Install [vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/), install
+[vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest) plugin with `vagrant plugin install vagrant-vbguest`.
+Simply call `vagrant up` to get your virtual machine running and `vagrant ssh` to jump into the virtual machine.
+
+Next steps:
+```
+indico admin user_create -a
```

Comment:
Yes, but `vagrant up` is about setting up the machine and provisioning it. The shell script itself is executed in the VM, while `vagrant up` is executed on the host. So you cannot use interactive scripts here unfortunately. Usually projects use default users (e.g., admin@example.com, admin) for development environments. We could maybe create such user with an SQL comannd, since Indico does not support non-interactive user creation.